pkg/variantregistry/snapshot.yaml

```diff
@@ -4319,7 +4315,7 @@ periodic-ci-jboss-eap-qe-openshift-eap-tests-pit-7.4.x-eap-ocp4.17-lp-interop-ea
     Aggregation: none
     Architecture: amd64
     CGroupMode: v2
-    ContainerRuntime: runc
+    ContainerRuntime: crun
```

Comment:
This doesn't look right, crun went default in 4.18, this job claims to be a 4.17 and does not explicitly claim crun, any idea why this flipped?

Reply:
It's because of this:

```
    Release: "7.4"
```

These jobs are misconfigured because they don't have the release label, so we fallback to making a guess based on what's in the job name.

I don't know there's much I can do it about this, I tried to get the release from Sippy, which comes authoritatively via the release label, otherwise it falls back.

Reply:
I assume there's no consistent rule that could be followed to instead grab the 4.17? It's just a shame to go from correct to not. But then again these are not jobs we deal with.

Reply:
The PR on balance makes it more correct than not, because we're now extracting the release from the sippy config when we can.  There's a few outliers like this one that become accidentally incorrect